### PR TITLE
Integrated featured roles into home page

### PR DESCRIPTION
**Pull request by kepeket**
Add featured roles, using the query engine and the same template as the search results

### CODE REVIEW

File: zubbi/views.py [hunk 1]
```diff
@@ -86,7 +86,21 @@ class IndexView(ZubbiMethodView):
     template_name = "index.html"
 
     def get(self):
-        context = self.get_context()
+        extra_filter = Q("term", private=False)
+        featured_roles = ["tox", "docker-image", "upload-artifactory"]
```
Comment (aladjadj) [inline zubbi/views.py]: can you move "star" roles to configuration file ?

**Review by aladjadj — CHANGES_REQUESTED** [review]
You can not harcoded roles because is specific to configuration repositories

File: zubbi/views.py (hunk 1 above)
Comment (kepeket) [reply zubbi/views.py]: done

**Comment by aladjadj** [comment]
Can you update the README please ?

**Review by aladjadj — APPROVED** [review]
LGTM

**Pull request merged**